packages/trip-form/src/styled.js

```diff
@@ -0,0 +1,135 @@
+import styled from "styled-components";
+
+// ////////////////////////////////////////////////
+// /////////////// App components /////////////////
+// ////////////////////////////////////////////////
+
+export const SettingsHeader = styled.div``;
+
+export const SettingsSection = styled.div``;
+
+export const SettingLabel = styled.label``;
+
+export const FloatingSettingLabel = styled(SettingLabel)`
+  float: left;
+`;
+
+export const DateTimeSelector = styled.div``;
+
+DateTimeSelector.DepartureRow = styled.div`
+  box-sizing: border-box;
+  > * {
+    box-sizing: border-box;
+    width: 33.333333%;
+    padding: 0px 5px;
+  }
+`;
+
+DateTimeSelector.DateTimeRow = styled.div`
+  box-sizing: border-box;
+  > * {
+    box-sizing: border-box;
+    width: 50%;
+    padding: 0px 5px;
+    display: inline-block;
+  }
+  input {
+    box-sizing: border-box;
+    width: 100%;
+  }
+`;
+
+export const ModeSelector = styled.div``;
+
+ModeSelector.MainRow = styled.div`
+  padding: 0px 5px;
+  box-sizing: border-box;
+  > * {
+    width: 100%;
+  }
+`;
+
+ModeSelector.SecondaryRow = styled.div`
+  > * {
+    width: 33.333333%;
+    padding: 0px 5px;
+  }
+`;
+
+ModeSelector.TertiaryRow = styled.div`
+  > * {
+    width: 33.333333%;
+    padding: 0px 5px;
+  }
+`;
+
+export const SubmodeSelector = styled(SettingsSection)``;
+
+SubmodeSelector.Row = styled.div``;
+
+SubmodeSelector.InlineRow = styled(SubmodeSelector.Row)`
+  text-align: right;
+`;
+
+// ////////////////////////////////////////////////
+// ///////////// Generic components ///////////////
+// ////////////////////////////////////////////////
```

Comment:
All the components below seem to also be components directly related to the app. I'd remove this comment. Eventually, we should figure out a better approach so that we're at least reusing some common button components or being better about making components that'd work with styled-system or whatever.

Reply:
I removed those carry over comments from one of the styled.js you wrote that I copied.